packages/core/src/__tests__/memdb.test.ts

```diff
@@ -65,6 +69,10 @@ describe('memdb', () => {
       kind: 1
     })
     expect(result.length).toBe(0)
+    const multipleParam = await model.findAll(core.class.Doc, {
```

Comment:
@BykhovDenis  @z2sx We have some troubles here in case we do search for core.class.Doc, 
assume even for Tx/Memdb stores we should search for all stores and do combine. And right now I think it will not work since it will choose a domain and do search.

Assume we have more domains, some in mongo, some in elastic it will trigger a full search.
Should we allow this?

Reply:
Could work for test